org-roam-compat.el

```diff
@@ -33,7 +33,89 @@
 ;;; Code:
 ;;;; Library Requires
 
+;;; Backports
+(when (< emacs-major-version 27)
+  ;; Emacs 26 does not have FOLLOW-SYMLINKS in `directory-files-recursively'
+  (defun directory-files-recursively (dir regexp
```

Comment:
While harmless in this case, I don't think it's a good idea to shadow the function like this. i.e. we should leave the `org-roam-` prefix here.